### ISSUE TITLE
Sugestion- Harder!

### ISSUE THREAD

**Issue by CTH999** [brandon3055/Draconic-Evolution#1046]
Suggestion 1- Once the chaotic tier is fully implemented, make armor, tools, weapons, so on, for it!

Suggestion 2- The chaotic core is way to easy to get, There should be a tier above it, a tier that's much, much, much harder! A.K.A. Requiring multiple chaotic cores to make!

Suggestion 3- An efficiency module for the armor! That way, you can use less power, but It can only be applied to chaotic+ Tools/ Armor!

**Comment by Saklad5** [comment]
The power consumption is the only thing providing a semblance of balance. The only thing that takes too much power, in my opinion, is the Shock Wave option on Draconic Bows. And to be honest, that’s not exactly underpowered anyway.

**Comment by CTH999** [comment]
True, but it would be maybe -1%, meaning that it's not much. It's a little, but not much.
